src/main/scala/rocket/CSR.scala

```diff
@@ -419,11 +419,14 @@ class CSRFile(
   val reg_vxsat = usingVector.option(Reg(Bool()))
   val reg_vxrm = usingVector.option(Reg(UInt(io.vector.get.vxrm.getWidth.W)))
 
-  val reg_instret = WideCounter(64, io.retire)
-  val reg_cycle = if (enableCommitLog) reg_instret else withClock(io.ungated_clock) { WideCounter(64, !io.csr_stall) }
+  val reg_mcountinhibit = RegInit(0.U((3 + nPerfCounters).W))
```

Comment:
Minor nit, use `CSRs.firstHPM` instead of the constant 3 so it's clear what the 3 represents.